src/badge/accolade/_accolade-badges.ts

```diff
@@ -154,7 +154,6 @@ export const AccoladeBadges: IBadgeData[] = [
     Watchman,
     HellAndBack,
     Traveler,
-    Vigilant,
```

Comment:
It looks like you removed Vigilant but forgot to add it again?

Reply:
Oh, nevermind... it is there, just not in the change list.

Reply:
Vigilant was in twice, which put it in the wrong place.